packages/common/src/sites/index.ts

```diff
@@ -1,41 +1,11 @@
-import { IHubRequestOptions } from "../types";
-import { stripProtocol } from "../urls";
-import { isGuid } from "../utils";
-import { lookupDomain } from "./domains";
-import { getSiteById } from "./get-site-by-id";
-
-/**
- * Returns site model given various kinds of identifier
- *
- * @param identifier - a site item ID, site hostname, enterprise site slug, or full site URL
- * @param hubRequestOptions
- */
-export function fetchSite(
-  identifier: string,
-  hubRequestOptions: IHubRequestOptions
-) {
-  let prms;
-
-  if (isGuid(identifier)) {
-    prms = getSiteById(identifier, hubRequestOptions);
-  } else {
-    let hostnameOrSlug = identifier;
-
-    // get down the the hostname
-    hostnameOrSlug = stripProtocol(hostnameOrSlug);
-    hostnameOrSlug = hostnameOrSlug.split("/")[0];
-
-    prms = lookupDomain(hostnameOrSlug, hubRequestOptions).then(({ siteId }) =>
-      getSiteById(siteId, hubRequestOptions)
-    );
-  }
-
-  return prms;
-}
-
+export * from "./_ensure-telemetry";
 export * from "./domains";
 export * from "./drafts";
-export * from "./_ensure-telemetry";
+export * from "./fetchSiteModel";
 export * from "./get-site-by-id";
+export * from "./HubSiteManager";
+export * from "./HubSites";
+export * from "./registerSiteAsApplication";
 export * from "./site-schema-version";
+export * from "./themes";
```

Comment:
ditto w/ all these - if we don't have a known place where we are going to call any of them from the Hub app, or hub-components, let's not export them. We can always export them later if needed. If we needlessly export them now it becomes very difficult to modify or remove them and adds to our maintenance costs.

Reply:
either needed directly or re-exported in hub-sites